bot/seasons/evergreen/snakes/utils.py

```diff
@@ -708,3 +705,12 @@ def _board_coordinate_from_index(self, index: int):
         if is_reversed:
             x_level = 9 - x_level
         return x_level, y_level
+
+    def _mod_in_game_check(self, user: Member) -> bool:
+        """Return True if user is a Moderator and playing the currently running game."""
+        return all((self._is_moderator(user), user in self.players))
```

Comment:
```suggestion
        return self._is_moderator(user) and user in self.players
```

The suggestion on line 639 would leave this function being used only on line 439. I don't think this simple condition warrants its own function in such case. Furthermore, based on the comments on line 439, the suggested simplification, which is based on the original code, may not exhibit behaviour which was the original intent anyway.

Reply:
This function may be useful after all. The suggested change on 619 I believe is the same condition that would be needed on line 439:

```suggestion
        return self._is_moderator(user) and user not in self.players
```

Reply:
Will fix this & the other logic issues tonight when I get home